sql.wit.md

```diff
@@ -0,0 +1,66 @@
+# `wasi-sql` API
+
+## Interfaces
+
+```wit
+interface "wasi:sql" {
+    // iterator item type
+    resource row {
+        field-name: list<string>
+        values: list<data-types>
+        index: u32
+    }
+    
+    // common data types
+    variant data-types {
+        integer(s32),
```

Comment:
I realise this is an early draft (and I may have rushed you, sorry) but presumably there will also be signed integers, long and short, etc.?

Some databases support unlimited-size integers I think - do we want to support those?

Reply:
done ✅